File: Python/PedraPapelTesoura.py
import random

opcoes = ["pedra", "papel", "tesoura"]

usuario = input("Escolha pedra, papel ou tesoura: ").lower()
computador = random.choice(opcoes)

print("Computador escolheu:", computador)

if usuario == computador:
    print("Empate!")

elif (usuario == "pedra" and computador == "tesoura") or \
     (usuario == "tesoura" and computador == "papel") or \
     (usuario == "papel" and computador == "pedra"):
    print("Você venceu!")

else:
    print("Computador venceu!")

File: Java/PedraPapelTesoura.java
import java.util.Random;
import java.util.Scanner;

public class PedraPapelTesoura {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        Random rand = new Random();

        String[] opcoes = {"pedra", "papel", "tesoura"};

        System.out.println("Escolha pedra, papel ou tesoura:");
        String usuario = sc.nextLine();

        String computador = opcoes[rand.nextInt(3)];

        System.out.println("Computador escolheu: " + computador);

        if(usuario.equals(computador)){
            System.out.println("Empate!");
        }
        else if(
            (usuario.equals("pedra") && computador.equals("tesoura")) ||
            (usuario.equals("tesoura") && computador.equals("papel")) ||
            (usuario.equals("papel") && computador.equals("pedra"))
        ){
            System.out.println("Você venceu!");
        }
        else{
            System.out.println("Computador venceu!");
        }

        sc.close();
    }
}